static/translations/messages_en.json

```diff
@@ -199,5 +199,6 @@
   "Reactions": "Reactions",
   "No reactions": "No reactions",
   "See who reacted": "See who reacted",
-  "Only organization admins are allowed to post to this stream.": "Only organization admins are allowed to post to this stream."
+  "Only organization admins are allowed to post to this stream.": "Only organization admins are allowed to post to this stream.",
+  "Copied email address": "Copied email address"
```

Comment:
A better string might be "Stream email copied to clipboard" or simply "Copied to clipboard".

Reply:
> Unfortunately, this change introduces a bug that causes the app to crash when you view the stream info for an unsubscribed stream. I think this is because stream email is unavailable for such streams, but you should investigate that further.

Yes the bug was indeed caused because of retrieving email address of an unsubscribed streams. Now I've put a check on it and it renders only if the stream is subscribed.

Now there are two options here 
1. We can show a message to user maybe in red, stating that he needs to subscribe to the stream to see the email address (like the image below) OR 
2. We won't render the email address view if stream is not subscribed.

<img src="https://user-images.githubusercontent.com/22986282/77796420-3c295080-7095-11ea-985d-5ee3809b62f9.png" height=450 width=250 />

Reply:
Definitely not red. That usually signals an actual error; it's not appropriate for something that's part of normal operation, and that many users won't need or want to change. Semantically, this seems to call for some flavor of grey.

Option 2 seems like it'll confuse users who *are* looking for the email address. (Or even users that aren't, and who are just exploring the stream-settings cards.)

Reply:
@ray-kraesig Sure will work on making it grey.